src/projectM-emscripten/README

```diff
@@ -1,27 +1,39 @@
 Running on the web.
 
-# Easy way:
+# TLDR:
+* `./autogen.sh`
+* `emconfigure ./configure --enable-emscripten --enable-gles --enable-sdl`
```

Comment:
i tried making it so all you need is `--enable-emscripten` now and it infers the others